src/SFA.DAS.EmployerFinance/Application/Commands/UpdateAccount/UpdateAccountCommand.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using MediatR;
+
+namespace SFA.DAS.EmployerFinance.Application.Commands.UpdateAccount
+{
+    public class UpdateAccountCommand : IRequest
```

Comment:
I didn't call it UpdateAccountName, in case we wanted to use the same command later to update other properties on Account, but happy to rename it.

Done.